### DISCUSSION TITLE
Asistencia

### DISCUSSION THREAD

**Discussion by AlessandrohL** [Launch-X-Latam/MisionFrontEnd#97] [Q&A] [answered]
Hola buenas tardes, tengo una duda y es si en alguna parte se debe de marcar asistencia?

**Comment by Amhedriel** [comment, marked as answer]
Buenas,

En el video anterior dijeron que no tomarían asistencia como tal, explicaron cómo se trabajaría con respecto a eso, te recomiendo que revises ese video y estés preparado para el live de hoy.

Saludos
